### ISSUE TITLE
[QUESTION] importing

### ISSUE THREAD

**Issue by gilcu3** [4f77616973/Wristkey#40]
How to currently import a file? I am using the adb method, but on WearOS 3 (therefore no FilePicker). 
For some reason the wifi is not recognized by this app, even though it works for adb. 
Also, to make this work the app must be debuggable, is there any known way to overcome that?

**Comment by 0x4f53** [comment]
> How to currently import a file? I am using the adb method, but on WearOS 3 (therefore no FilePicker).
https://github.com/0x4f53/Wristkey/wiki/Transferring-Data

> For some reason the wifi is not recognized by this app, even though it works for adb.
Please make sure both devices are on the same Wi-Fi network for this.

> Also, to make this work the app must be debuggable, is there any known way to overcome that?
You can clone it and build it manually via gradle.

**Comment by gilcu3** [comment]
> > How to currently import a file? I am using the adb method, but on WearOS 3 (therefore no FilePicker).
> 
> https://github.com/0x4f53/Wristkey/wiki/Transferring-Data
> 
> > For some reason the wifi is not recognized by this app, even though it works for adb.
> 
> Please make sure both devices are on the same Wi-Fi network for this.

I did exactly that, but the wear os app could not detect the wifi connection.

**Comment by 0x4f53** [comment]
Can you send me a logcat then?

**Comment by gilcu3** [comment]
Yes, I should have done that from the beginning. This [log.txt](https://github.com/0x4f53/Wristkey/files/14570406/log.txt) comes from a debug build I did myself, in sync with the latest commit here. I already have some accounts working (setup using adb method). I just pressed the wifi button and got the cannot detect wifi sign, while at the same time I am actually capturing the logcat through adb wifi.

**Comment by gilcu3** [comment]
> well the adb debug method is really only for devices without wifi (only one that comes to mind). the main way is now via wifi.

Got it, it makes sense to deprecate that feature. That makes it even more important to figure out why Wifi is not detected on the Ticwatch

**Comment by 0x4f53** [comment]
@gilcu3 Any luck with this? I don't have a TicWatch with me so I cannot figure this one out. The logcat doesn't seem to have anything unusual either.

**Comment by gilcu3** [comment]
No, unfortunately, I have been using the app with my own debug build to workaround the issue.
